docs/releasing.md

```diff
@@ -0,0 +1,128 @@
+# Releasing ironic-image
+
+This document explains how to prepare and publish a release for ironic-image, and
+how often a new release branch is created.
+
+**NOTE**: Always follow
+[release documentation from the main branch](https://github.com/metal3-io/ironic-image/blob/main/docs/releasing.md).
+Release documentation in release branches may be outdated.
+
+## Before making a release
+
+Things you should check before making a release:
+
+- Check the
+  [Metal3 release process](https://github.com/metal3-io/metal3-docs/blob/main/processes/releasing.md)
+  for high-level process and possible follow-up actions
+- Verify the latest bgufix branches in ironic and ironic-inspector upstream repositories;
+- Verify compatibility with latest sushy and ironic-lib upstream releases or master
+  branches based on ironic and ironic-inspector bugfix requirements and constraints
+- Verify openstack upper-constraints compatibility with ironic and ironic-inspector
+  bugfix branches
+- Verify any other direct or indirect dependency is uplifted to close any public
+  vulnerabilities
+
+  ## Permissions
+
+Creating a release requires repository `write` permissions for:
+
+- Tag pushing
+- Branch creation
+- GitHub Release publishing
+
+These permissions are implicit for the org admins and repository admins.
+Release team member gets his/her permissions via `metal3-release-team`
+membership. This GitHub team has the required permissions in each repository
+required to release CAPM3. Adding person to the team gives him/her the necessary
+rights  in all relevant repositories in the organization. Individual persons
+should not be given permissions directly.
+
+## Process
+
+ironic-image uses [semantic versioning](https://semver.org). For version `vX.x.y`:
+
+### Repository setup
+
+Clone the repository:
+`git clone git@github.com:metal3-io/ironic-image`
+
+or if using existing repository, verify your intended remote is set to
+`metal3-io`: `git remote -v`. For this document, we assume it is `origin`.
+
+- If creating a new release branch, identify the commit you wish to create the
+  branch from, and create a branch `release-X.x` where `X.x` corresponds to the
+  ironic upstream bugfix version, for example for `bugfix-22.1` it's `release-22.1`:
```

Comment:
nit: depending on the exact moment, it can also be a stable branch

Reply:
I suggest we don't do things like `release-<codename>` in this case and keep the number (corresponding to Ironic's release number)